docs/contracts.rst

```diff
@@ -23,7 +23,10 @@ name as the contract) is executed once.
 From ``web3.js``, i.e. the JavaScript
 API, this is done as follows::
 
+    var source = <<Need to specify some source including contract name for the data param below.>>
```

Comment:
What about `var source = "contract CONTRACT_NAME {}"; // Need to specify some source including contract name for the data param below.` - then it works as is.